### PR TITLE
Add extra queries and update table

### PR DESCRIPTION
**Pull request by lucabenaventew**
# Project Updates :green_circle: 
image
image
image
image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -8,3 +8,11 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered IS true;
 SELECT * FROM animals WHERE NOT name = 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.3 AND 17.4;
+SELECT COUNT(*) FROM animals;
```
Comment (tufoinnkuo10) [inline queries.sql:11]: - [ ] please ensure to add steps when you `UPDATE` the animals' table by setting the species column to unspecified, then roll back the change
- [ ] Also steps when you commit a transaction, delete all records in the animals table, then roll back the transaction

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @lucabenaventew 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message


Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by lucabenaventew** [comment]
# Project Updates :green_circle: 
- [ ] Add steps
image
image
image
image
image

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Luca,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾  Congratulations!  🎉💯🌟 

## Highlights 👍
- Database queries properly implemented

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You can also consider:**
- This project appears to be worked on by a sole contributor, therefore kindly update the Author section of the ReadMe accordingly. 
- MIT license link in your ReadMe is broken, kindly endeavor to fix it.
Implementing the above change(s) will make your ReadMe/ repository stand out even better 😉

Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**